osmaxx/api_client/API_client.py

```diff
@@ -4,6 +4,8 @@
 
 logger = logging.getLogger(__name__)
 
+HTTPError = requests.HTTPError
```

Comment:
That intention would be much clearer to me, if we defined our own class in `API_client.py` and would document this as eg. `thin wrapper` around the requests HTTPError. Like it stands currently, I would assume I can actually remove the declaration as it isn't used.